content/tbd/1000 Landscapes for 1 billion people.md

```diff
@@ -0,0 +1,53 @@
+---
+created: 2023-11-30
+contributors:
+  - Catherine Tran
+title: 1000 Landscapes for 1 Billion People
+url: https://landscapes.global/
+locations:
+  - International network
+start: 2018
+end: 
+size: 4-10
+image: assets/img/1000-landscapes-for-1-billion-people-homepage.png
+sectors: 
+activities: 
+cohere: true
+metadata_version: "1"
+---
+
+# About
+
+Who are they? What do they do? What is unique, special, or distinctive about them?
```

Comment:
This is why i think we don't want this stuff in the template because we just end up with it hanging around accidentally in the page when we insert the template.